ssh/tailssh/incubator.go

```diff
@@ -184,6 +186,7 @@ func parseIncubatorArgs(args []string) (a incubatorArgs) {
 	flags.BoolVar(&a.isShell, "shell", false, "is launching a shell (with no cmds)")
 	flags.BoolVar(&a.isSFTP, "sftp", false, "run sftp server (cmd is ignored)")
 	flags.StringVar(&a.loginCmdPath, "login-cmd", "", "the path to `login` cmd")
+	flags.StringVar(&a.homeDir, "home-dir", "/", "the path to user `$HOME` directory")
```

Comment:
there's nothing really "home" about this from the incubator's perspective. This is just the dir it uses as its pwd. The fact that it might've come from $HOME originally is just a detail for the incubator. How about we just rename it pwd? I can do that for you.